libs/CMakeLists.txt

```diff
@@ -97,11 +105,13 @@ macro(add_external_project_impl name)
       CMAKE_ARGS -DCMAKE_BUILD_TYPE=Release
       -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
       -DCMAKE_INSTALL_PREFIX=${source_dir}
-      $<$<BOOL:KW_CMAKE_PREFIX_PATH>:"-DCMAKE_PREFIX_PATH=${KW_CMAKE_PREFIX_PATH}">
+      $<$<BOOL:KW_CMAKE_PREFIX_PATH>:-DCMAKE_PREFIX_PATH="${KW_CMAKE_PREFIX_PATH}">
       ${KW_CMAKE_ARGS}
       INSTALL_COMMAND ${KW_INSTALL_COMMAND}
       BUILD_COMMAND ${KW_BUILD_COMMAND}
-      BUILD_BYPRODUCTS ${KW_BUILD_BYPRODUCTS})
+      BUILD_BYPRODUCTS ${KW_BUILD_BYPRODUCTS}
+      BYPRODUCTS ${KW_BYPRODUCTS}
```

Comment:
From the documentation seems like `BUILD_BYPRODUCTS` is what we should use. This then passes it on to `BYPRODUCTS`

Reply:
```
BUILD_BYPRODUCTS <file>...
New in version 3.2.

Specifies files that will be generated by the build command but which might or might not have their modification time updated by subsequent builds. This may also be required to explicitly declare dependencies when using the [Ninja](https://cmake.org/cmake/help/latest/generator/Ninja.html#generator:Ninja) generator. These ultimately get passed through as BYPRODUCTS to the build step's own underlying call to [add_custom_command()](https://cmake.org/cmake/help/latest/command/add_custom_command.html#command:add_custom_command), which has additional documentation.
```